pkg/grub/rootfs.cfg

```diff
@@ -158,9 +158,9 @@ function set_generic {
       set_global hv_eve_mem_settings "eve_mem=6500M,max:6500M"
       set_global hv_ctrd_mem_settings "ctrd_mem=4000M,max:4000M"
```

Comment:
Shouldn't we apply the same approach in the kubevirt case?

Reply:
Good question. I'm sure how kubevirt works and what's the expectations for that setup. We can ask @andrewd-zededa to consult us here.